chord_diagrams.Rmd

```diff
@@ -0,0 +1,156 @@
+    
+  - *How many categories?* There are no limits on the number of groups that can be technically depicted on a chord diagram. But chord diagrams can become difficult to read and interpret when the number of groups is high. This is because many chords will be thinner and more difficult to disentangle. Adding interactivity can help by highlighting one sector at a time (see Interactivity using chorddiag below) but ideally, chord diagrams should contain about 10 or fewer categories. Adjustments might be made to the data beforehand to reduce the number of categories. For example, if you want to depict migration flow, instead of displaying migration between all countries in the dataset, you could group them into continents or regions or instead, display a subset of the countries. 
+  
+  - *What is the directionality of flow?* Chord diagrams can be directional or non-directional. Directional data will have two chords for each sector, a chord going from a given category out to a target and the other coming from that target. Non-directional or symmetric data on the other hand only have one value per chord (the same value for each direction). Adjustments in color can be made to the chord diagram depending on the directionality of the data.
+
+
+## The chordDiagram function
+
+To create chord diagrams we will use the chordDiagram function from the **circlize** package in R. The circlize package itself contains methods from the circos tool used for graphing circular representations, especially used in the field of Genomics. The chordDiagram help documentation describes how data should be arranged. We will summarize this briefly here: 
+
+chordDiagram takes a datafame of three columns:
+
+  - rn: The first column is the "from" column in an adjacency list where each row is a sector/track/category name. 
+  - cn: The second column is the "to" column in an adjacency list where each row is a sector/track/category name.  
+  - value: the values for the interaction or relation between columns 1 and 2, the number of counts between the two categories.
+
+## Basic Chord diagrams: Friends 
+
+  In this example, we will use chord diagrams to make a simple visualization of the relationships between six main characters from the popular TV show Friends. We will visualize dyadic (pairwise) storylines between characters in the show across all episodes. Note that only dyads will be considered so for example, in Season 1 Ep.1, a plotline involving Chandler Joey and Ross will not be included in our chord diagram. The relationships here are non-directional (or symmetrical) meaning if for example, Joey is in a plot line with Rachel then Rachel is in a plotline with Joey.
+  
+```{r Prepare the data}
```

Comment:
Please update to `{r} `instead of `{r Prepare the data}`. Similar comment for all the statements below.

Reply:
Done! committed new changes